tools/wallet/src/core/legacy_scheme.rs

```diff
@@ -42,3 +42,36 @@ impl LegacyKeyScheme {
         LegacyKeyScheme::new(&wallet)
     }
 }
+
+#[cfg(test)]
```

Comment:
I don't think this needs to be created as a `mod`. You can just move this to a test section, and import `use diem_crypto::Length;` with a cfg(tests) annotation.

Reply:
The test has been updated.